mbway/src/main/java/com/adyen/checkout/mbway/DefaultMBWayDelegate.kt

```diff
@@ -61,11 +75,20 @@ internal class DefaultMBWayDelegate(val paymentMethod: PaymentMethod) : MBWayDel
         componentStateChanged(paymentComponentState)
     }
 
+    override fun getSupportedCountries(): List<CountryInfo> = CountryUtils.getCountries(SUPPORTED_COUNTRIES)
```

Comment:
We have some redundant stuff in `MBWayComponent` now, like `getSupportedCountries` and the companion `val`s. Should we remove them from there? (maybe just use `emptyList()` in `MBWayView` where we call `getSupportedCountries`, since that view will be removed soon anyway.